packages/config/src/projects/da-beat/index.test.ts

```diff
@@ -1,45 +1,23 @@
 import { expect } from 'earl'
 import { daLayers } from '.'
-import type { ProjectLinks } from '../../types'
 import { layer2s } from '../layer2s'
 import { layer3s } from '../layer3s'
 
 describe('DA-BEAT', () => {
-  daLayers.forEach((layer) => {
-    describe(layer.display.name, () => {
+  daLayers.forEach((project) => {
+    describe(project.display.name, () => {
       it('should contain description with dot at the end', () => {
-        expect(layer.display.description.endsWith('.')).toEqual(true)
+        expect(project.display.description.endsWith('.')).toEqual(true)
       })
 
-      layer.bridges.forEach((bridge) => {
-        describe(bridge.display.name, () => {
-          it(`should contain bridge description with dot at the end`, () => {
-            expect(bridge.display.description.endsWith('.')).toEqual(true)
+      project.daLayer.bridges.forEach((bridge) => {
+        if (bridge.display) {
+          it(`${bridge.display.name} description ends with a dot`, () => {
+            expect(bridge.display?.description.endsWith('.')).toEqual(true)
           })
-        })
+        }
       })
     })
-
-    describe(`${layer.display.name} does not have duplicated links`, () => {
```

Comment:
We don't want to have this test?

Reply:
It was testing for bridge having links duplicated from da layer. Bridges don't have separate links now